build/Build.cs

```diff
@@ -0,0 +1,1045 @@
+            ControlFlow.Assert(FileExists(workingDirectory / rpmARM32PackageFilename), $"Missing {rpmARM32PackageFilename}");
+            ControlFlow.Assert(FileExists(workingDirectory / rpmx64PackageFilename), $"Missing {rpmx64PackageFilename}");
+
+            OctoCliTool($"pack --id=Octopus.Tentacle.CrossPlatformBundle --version={OctoVersionInfo.FullSemVer} --basePath={workingDirectory} --outFolder={ArtifactsDirectory / "nuget"}");
+        });
+
+    // ReSharper disable once UnusedMember.Local
+    Target PackWindows => _ => _
+        .Description("Packs all the Windows targets.")
+        .DependsOn(BuildWindows)
+        .DependsOn(PackWindowsZips)
+        .DependsOn(PackChocolateyPackage)
+        .DependsOn(PackWindowsInstallers);
+
+    // ReSharper disable once UnusedMember.Local
+    Target PackLinux => _ => _
+        .Description("Packs all the Linux targets.")
+        .DependsOn(BuildLinux)
+        .DependsOn(PackLinuxTarballs)
+        .DependsOn(PackDebianPackage)
+        .DependsOn(PackRedHatPackage);
+
+    // ReSharper disable once UnusedMember.Local
+    // ReSharper disable once InconsistentNaming
+    Target PackOSX => _ => _
+        .Description("Packs all the OS/X targets.")
+        .DependsOn(BuildOSX)
+        .DependsOn(PackOSXTarballs);
+
+    Target Pack => _ => _
+        .Description("Pack all the artifacts. Notional task - running this on a single host is possible but cumbersome.")
+        .DependsOn(PackCrossPlatformBundle);
+    #endregion
+    
+    #region tests
+    // ReSharper disable once UnusedMember.Local
+    Target TestWindows => _ => _
+        .DependsOn(BuildWindows)
+        .Executes(RunTests);
+    
+    // ReSharper disable once UnusedMember.Local
+    Target TestLinux => _ => _
+        .DependsOn(BuildLinux)
+        .Executes(RunTests);
+    
+    // ReSharper disable once UnusedMember.Local
+    Target TestOSX => _ => _
+        .DependsOn(BuildOSX)
+        .Executes(RunTests);
+
+    Target TestLinuxPackages => _ => _
+        .Description("Tests installing the .deb and .rpm packages onto all of the Linux target distributions.")
+        .DependsOn(PackDebianPackage)
+        .DependsOn(PackRedHatPackage)
+        .Executes(() =>
+        {
+            void RunLinuxPackageTestsFor(IReadOnlyList<string> testConfiguration)
+            {
+                var framework = testConfiguration[0];
+                var runtimeId = testConfiguration[1];
+                var dockerImage = testConfiguration[2];
+                var packageType = testConfiguration[3];
+
+                InTest($"{framework}/{runtimeId}/{dockerImage}/{packageType}", () =>
+                {
+                    string? archSuffix = null;
+                    if (packageType == "deb")
+                    {
+                        if (runtimeId == "linux-x64") archSuffix = "_amd64";
+                    }
```

Comment:
we can shrink this down
```suggestion
                    if (packageType == "deb" && runtimeId == "linux-x64") archSuffix = "_amd64";
```